<launch>
    <node name="monocular_depth_node" pkg="robohop_ros" type="monocular_depth.py" output="screen"/>
</launch>
